docs/api.md

```diff
@@ -39,9 +39,9 @@ https://api.regulations.gov/v4/END_POINT?filter[lastModifiedDate][ge]=2020-08-10
 ### **What the client should do**
 While the above section describes how to generate all jobs, the client will do the actual work for each job. 
 
-This will be relatively simple: return the JSON given by the endpoint to the work server.
+The client will get a job from regulations.gov, process the job URL, and finally downloads the job.
 
-Additionally, if a document/comment is being retrieved, return any attachments that go along with that document/comment. These can be identified in the JSON given under data/attributes/fileFormats.
+Additionally, if a document/comment is being retrieved, it will return any attachments that go along with that document/comment. These can be identified in the JSON given under data/attributes/fileFormats.
```

Comment:
You may want to include that the client will also turn around and download these attachments.